tailwind.config.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
     extend: {
       colors: {
         primary: "#f02e65",
+        onhover: "#e00b41",
```

Comment:
Can you take a color which is in a range of pink. Otherwise it is looking good 😄